yang/vyatta-protocols-frr-isis-v1.yang

```diff
@@ -0,0 +1,822 @@
+    ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+    POSSIBILITY OF SUCH DAMAGE.
+
+    SPDX-License-Identifier: BSD-3-Clause
+    The YANG module package for vyatta-protocols-frr-isis-v1";
+
+  revision 2018-11-15 {
+    description "Initial revision of version 1";
+  }
+
+
+  typedef level {
+    type enumeration {
+      enum "level-1" {
+        value 1;
+        configd:help "This enum indicates L1-only capability.";
+        description "This enum indicates L1-only capability.";
+      }		
+      enum "level-2" {
+        value 2;
+        configd:help "This enum indicates L2-only capability.";
+        description "This enum indicates L2-only capability.";
+      }
+      enum "level-1-2" {
+        value 3;
+        configd:help "This enum indicates capability for both levels.";
+        description "This enum indicates capability for both levels.";
+      }
+    }
+    configd:help "This type defines IS-IS level of an object.";
+    description "This type defines IS-IS level of an object.";
+  } 
+
+  typedef extended-circuit-id {
+    type uint32;
+    configd:help "This type defines the extended circuit ID associated with an interface.";
+    description "This type defines the extended circuit ID associated with an interface.";
+  }
+
+  typedef network-type {
+    type enumeration {
+      enum "unknown" {
+        value 0;
+        configd:help "Unknown network type. Only valid as a state.";
+        description "Unknown network type. Only valid as a state.";
+      }
+      enum "broadcast" {
+        value 1;
+        configd:help "Broadcast circuit network-type.";
+        description "Broadcast circuit network-type.";
+      }
+      enum "point-to-point" {
+        value 2;
+        configd:help "Point-to-point circuit network-type."; 
+        description "Point-to-point circuit network-type.";
+      }
+      enum "loopback" {
+        value 3;
+        configd:help "Loopback circuit network-type. Only valid as a state.";
+        description "Loopback circuit network-type. Only valid as a state.";
+      }
+    }
+  }
+
+  typedef lsp-id {
+    type string {
+      pattern "[0-9A-Fa-f]{4}\\.[0-9A-Fa-f]{4}\\.[0-9A-Fa-f]{4}\\.[0-9][0-9]-[0-9][0-9]";
+    }
+    configd:help  "This type defines the IS-IS LSP ID format using a 
+    pattern, An example LSP ID is 0143.0438.AeF0.02-01"; 
+    description "This type defines the IS-IS LSP ID format using a
+    pattern, An example LSP ID is 0143.0438.AeF0.02-01";
+  }
+
+  typedef snpa {
+    type string {
+      length "0 .. 20";
+    }
+    configd:help "This type defines the Subnetwork Point of Attachment (SNPA) format.
+    The SNPA should be encoded according to the rules specified for the particular type
+    of subnetwork being used. As an example, for an ethernet subnetwork, the SNPA is 
+    encoded as a MAC address like '00aa.bbcc.ddee'.";
+
+    description"This type defines the Subnetwork Point of Attachment (SNPA) format.
```

Comment:
Please add a space after the `description` keyword. This prevents a `pyang` error:
```
vyatta-protocols-frr-isis-v1.yang:148: error: syntax error: expected separator, got: ""This ..."
```

Reply:
Actually it looks like this typedef is unused? It could be removed.

Reply:
This is Resolved.